orcestra_book/data_policy.md

```diff
@@ -0,0 +1,13 @@
+# Data Policy
+
+ORCESTRA intends to follow the open data policy established for NARVAL. This means that we do not intend to police the use of, or access to, the data. Instead we rely on the good will and scientific integrity of those interested in using the data to respect the efforts of those whose were responsible for its collection by adhering to the two guidelines below:
+
+1. No redistribution: Groups who are using the data are asked not to redistribute this data, but rather refer those interested in accessing the data to the original source, and these guidelines for the data use.
+
+2. Coordinated Use and Acknowledgement: Unlike in some other contexts the ORCESTRA data is made available through the individual efforts and particular funding of a number of independent research groups. Often the data that is collected represents the fruits of many years of effort to develop, certify and operate an instrument. This effort is often connected to individual research projects, and even PhD projects. For this reason, we consider it a matter of scientific integrity for users to respect the priority of those who collected the data, by coordinating your planned use with the instrument PIs (list will follow), and making an effort to explicitly and clearly acknowledge the origin of the data when it is presented to a broader community in the form of publications or presentations.
+
+3. For publications that include ORCESTRA data, include the following in your acknowledgements: The data used in this publication was gathered in the ORCESTRA field campaign and is made available through [insert name of the institute]. ORCESTRA is funded with support of the European Space Agency (ESA), the Max Planck Society (MPG), the German Research Foundation (DFG) and the German Aerospace Center (DLR).
+
+4. In addition, specific data may be bound by agreements associated with the platforms from which they were collected, for instance the data policies of the [German Research Vessels](), or [HALO](https://halo-research.de/data/).
```

Comment:
The build current doesn't work because there's a link to `German Research Vessels` missing.

Reply:
... also the HALO-link doesn't seem to point to a "data policy"

Reply:
Yeah... I've read through the website and there is no clear section pointed to data policy. I think the data policy is somehow intertwined in the text (or I interpreted it wrong...)

Reply:
I'd suggest to change the formulation to this one:
```suggestion
4. In addition, specific data may be bound by agreements associated with the platforms from which they were collected.
```
or
```suggestion
```
We should not bind us to things which we don't know. We also want to have a compact and readable policy, thus it might be sensible to not expand the effective size of the policy by including all the other things.